src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/location/LocationType.kt

```diff
@@ -10,6 +10,7 @@ enum class LocationType(val label: String) {
   CC("Crown Court"),
   CM("Combined Court"),
   CO("County Court"),
+  CRT("Court"),
```

Comment:
This new location type is to support the mapping of courts where we cannot work out the type based on the name of the location (see code changes further down).  It will then be up to the end users to correct this via the front end.